### ISSUE TITLE
icloudpd/icloudpd:latest (1.8.0) throws "keyring.errors.NoKeyringError" error

### ISSUE THREAD

**Issue by rweindl** [icloud-photos-downloader/icloud_photos_downloader#556]
After updating all containers of image icloudpd:icloudpd to version 1.8.0 I receive a keyring.errors.NoKeyringError.

## Steps to Reproduce
1.  Install icloudpd/icloudpd:latest (1.8.0) via Docker (on e.g. Synology NAS)
2. Execute either `icloud --username <USERNAME>` or `icloudpd. --username <USERNAME> --directory /data --recent 10` in container's Terminal

## Expected Behavior
- I can execute those commands

## Actual Behavior
Receiving following error:
```
2023-01-30 20:08:18 DEBUG    Authenticating...                                                               
Traceback (most recent call last):                                                                           
  File "icloudpd.py", line 5, in <module>                                                                    
  File "click/core.py", line 1130, in __call__                                                               
  File "click/core.py", line 1055, in main                                                                   
  File "click/core.py", line 1404, in invoke                                                                 
  File "click/core.py", line 760, in invoke                                                                  
  File "icloudpd/base.py", line 268, in main                                                                 
  File "icloudpd/authentication.py", line 29, in authenticate                                                
  File "pyicloud_ipd/base.py", line 141, in __init__                                                         
  File "pyicloud_ipd/utils.py", line 35, in get_password_from_keyring                                        
  File "keyring/core.py", line 55, in get_password                                                           
  File "keyring/backends/fail.py", line 25, in get_password                                                  
keyring.errors.NoKeyringError: No recommended backend was available. Install a recommended 3rd party backend 
package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://py
pi.org/project/keyring for details.
```

## Context
icloudpd command worked before I upgraded to latest 1.8.0 version. I ran into the same issue as reported in #539 before and so thought upgrading will fix that.

**Comment by AndreyNikiforov** [comment]
Known problem: #539

Fix is merged #554 and will be in the next version.

Mitigation: use 1.7.3 docker

**Comment by rweindl** [comment]
Thank you for the update @AndreyNikiforov! I missed the comment in #539 about the same are issue. Will wait for next version to retest.
